console/console-init/ui/src/components/MetadataListTable/TableRow.tsx

```diff
@@ -37,31 +38,20 @@ const styles = StyleSheet.create({
   }
 });
 
+interface IRowOptions {
+  key: string;
+  type: any;
```

Comment:
I think we can declare a type of 'type' here as String|Number|Array|Object ....

Reply:
yes,noted down.
We will take care later after completion of create metadata screen, same data types used on that screen . I will reuse that interface.